README.md

```diff
@@ -78,34 +86,26 @@ const response = await fetch(poetNodeUrl + '/identities/', {
 })
 ```
 
-
-Notice you don't need to wait for the server's response to know the claim's Id. You don't even need to publish it! `claim.id` is readily available right after calling `createClaim`.
+Notice you don't need to wait for the server's response to know the claim's ID. You don't even need to publish it! `claim.id` is readily available right after calling `createClaim`.
 
 ## Contributing
 
 ### Compiling
-Run `npm run build` to compile the source. This will run TypeScript on the source files and place the output in `dist/ts`, and then it'll run Babel and place the output in `dist/babel`.
 
-Currently, we're only using Babel to support [absolute import paths](https://github.com/tleunen/babel-plugin-module-resolver) in the unit tests. 
-> Absolute paths are only used in the tests. This is due to how Typescript and Babel process absolute paths: on build, Typescript transforms the .ts files into .js and places type definitions in .d.ts files. Babel, with the module-resolver plugin, will then transform the absolute paths in these .js files into relative paths, but will leave the .d.ts unchanged — which still have absolute paths. This causes issues with clients of the library that want to use these typescript definitions.   
+Run `npm run build` to compile the source. This will run TypeScript on the source files and place the output in `dist/ts`, and will then run Babel and place the output in `dist/babel`.
+
+Currently, we're only using Babel to support [absolute import paths](https://github.com/tleunen/babel-plugin-module-resolver) in the unit tests. This is due to how TypeScript and Babel process absolute paths. On build, TypeScript transforms the .ts files into .js and places type definitions in .d.ts files. Babel, with the module-resolver plugin, will then transform the absolute paths in these .js files into relative paths, but will leave the .d.ts unchanged, which still have absolute paths. This causes issues with clients of the library that want to use these TypeScript definitions.   
 
 ### Tests
-Run all tests with `npm test`.
 
+Run all tests with `npm test`.
 
 ### Coverage
-Coverage is generated with [Istanbul](https://github.com/istanbuljs/nyc) whenever tests are run.
-A more complete report can be generated by running `npm run coverage`.
 
-This area needs some reviewing — the reports look a bit off sometimes.
+Coverage reports are created with [Istanbul](https://github.com/istanbuljs/nyc), which can be generated by running `npm run coverage`.
 
-Also, we'll want the Travis job to check that coverage doesn't go down with pull requests. 
+<!-- TODO: This area needs some reviewing; the reports look a bit off sometimes. Also, we'll want the Travis job to check that coverage doesn't go down with pull requests. -->
```

Comment:
@warrenv @wzalazar maybe we can now complete remove this?

Reply:
I will remove it. @warrenv or @wzalazar can create a ticket for it if it is still an issue.